x-pack/platform/plugins/shared/fleet/server/routes/epm/handlers.ts

```diff
@@ -105,12 +106,25 @@ export const getListHandler: FleetRequestHandler<
   undefined,
   TypeOf<typeof GetPackagesRequestSchema.query>
 > = async (context, request, response) => {
-  const savedObjectsClient = (await context.fleet).internalSoClient;
+  const fleetContext = await context.fleet;
+  const savedObjectsClient = fleetContext.internalSoClient;
   const res = await getPackages({
     savedObjectsClient,
     ...request.query,
   });
   const flattenedRes = res.map((pkg) => soToInstallationInfo(pkg)) as PackageList;
+
+  if (request.query.withPackagePoliciesCount) {
+    const countByPackage = await getPackagePoliciesCountByPackageName(
```

Comment:
curious why not use `packagePolicyService.list` instead of a new function?

Reply:
that function does an aggregation in `elasticsearch` on package policy name as we do not need to return package policies, we could eventually update the list method to support passing aggregations but we will still probably need an extra function to abstract that aggregation wdyt?

Reply:
👍 I also missed that `getPackagePoliciesCountByPackageName` keys by package name, not that it returns a count based on a package name param (that's why I thought we could use `packagePolicyService.list` kuery + `total`). thanks for clarifying